ruby_on_rails/active_record_basics/active_record_basics.md

```diff
@@ -1,3 +1,10 @@
+<div class="lesson-note lesson-note--tip" markdown="1">
```

Comment:
Instead of these notices, I think we can do one of two things.

1. We could avoid changing the Odin site repo until all of your work with these lessons is complete. This way, people currently working through the material will just work with the original lesson up until everything is ready to go.
2. We could one-by-one remove the sections as you've finished them. So for this PR, you could remove the stuff about associations from the Active Record Basics lesson. Then we could merge in this PR and change the main app to immediately begin including the new Basic Associations lesson. This feels natural to me and is probably my preference for something like this.

Did  you have any thoughts on this @KevinMulhern ?

Reply:
If I may suggest while we await @KevinMulhern's response -- the only potential problem I see with one-by-one removal is that perhaps it's better for the student to read the documents in the originally-intended order. For that reason, I like your #1 solution the best.

Reply:
Co-sign no notices. Disruption should be minimal enough to not need them. There will only be a handful of students going through the later lessons in the curriculum at any one time compared to thousands in foundations.

Either option sounds great to me. First is probably best for students who have already completed the content and will want to go back and mark it as completed. Second is probably better for students going through that lesson right now since we're doing it back to front. I think I'd lean _just_ slightly towards the second for that reason.

Reply:
Thank you for your guys' time. How does this work -- shall I close this PR and open a second one that adheres to @JoshDevHub's suggestion to one-by-one remove?

Reply:
No need to close this PR.

Locally, you want to get back to your `Add_basic_associations_rails_lesson` branch. You can make new commits to that branch and after you push, they'll automatically be included here.

So you can just add a couple of commits for:
- Removing the notices at the top of the lessons.
- Removing the associations content from the Active Record Basics lesson.

Definitely let me know if any of that is unclear or you have any other questions :thumbsup:

Reply:
I have made the two additional commits.